Slam/DefaultAddMatchInteractor.swift

```diff
@@ -2,12 +2,20 @@ import Foundation
 
 public class DefaultAddMatchInteractor : AddMatchInteractor {
 
+    var api : SlamAPI?
     let view : AddMatchView
+
     public init(view: AddMatchView) {
```

Comment:
This constructor should probably be removed. I would update the existing tests that initialize this class to pass in a mock Slam API object.